Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -902,16 +902,8 @@ private void OnInternalChangingRole(ChangingRoleEventArgs ev)
                 if (Ammo.Count > 0)
                 {
                     ev.Ammo.Clear();
-                    Timing.CallDelayed(
```

Comment:
Delay necessary to preserve ammo values if they exceed the server's configured ammo limits. Doing it with ev.Ammo results in excess ammo being dropped on the ground below them, rather than overriding the ammo limits.

Reply:
that seems more about a needed fix in the code ?